name-registry/project/contracts/registry-contract/src/main.sw

```diff
@@ -152,4 +172,10 @@ impl Info for Contract {
             Option::None => Result::Err(RegistrationValidityError::NameNotRegistered),
         }
     }
+
+    #[storage(read)]
+    fn rate(id: ContractId) -> Option<u64> {
+        // SAFETY TODO: will bump in newer release PR
+        storage.assets.get(id).unwrap()
```

Comment:
This will fail if I pass an invalid contract id. There should be a check for `None` before unwrapping.